DXMainClient/Domain/Multiplayer/MapLoader.cs

```diff
@@ -114,6 +117,8 @@ private void LoadMultiMaps(IniFile mpMapsIni)
                 maps.Add(map);
             }
 
+            _translatedMapNames["Unknown"] = "Unknown".L10N("Client:Main:UnknownMap");
```

Comment:
What about the game mode? I would also just leave it at "Client:Main:Unknown" so that it is shared.

Reply:
Currently it's covered in `CnCNetLobby.cs` file:
```
            string translatedGameMode = hg.GameMode.L10N($"INI:GameModes:{hg.GameMode}:UIName", notify: false);
```

I can also write an if statement here if you think "Client:Main:Unknown" is better

Reply:
Won't it fail to parse the game mode before that though?

Reply:
oh right let me think

Reply:
And I think it's a special case, so it needs special handling. Notification for the translation system for sure.

Reply:
> Won't it fail to parse the game mode before that though?

`hg.GameMode` is a string